extensions/ql-vscode/src/vscode-tests/cli-integration/run-cli.test.ts

```diff
@@ -25,11 +25,13 @@ describe('Use cli', function() {
   });
 
   it('should have the correct version of the cli', async () => {
-    expect(
-      (await cli.getVersion()).toString()
-    ).to.eq(
-      new SemVer(process.env.CLI_VERSION || '').toString()
-    );
+    if (process.env.CLI_VERSION !== 'nightly') {
```

Comment:
Minor: It might be worth pushing the `if` statement up so we only run this test if we aren't on a nightly CLI version.  Currently if `CLI_VERSION === 'nightly'` this will suggest we've checked that the version of the CLI is correct when we haven't checked anything.

Reply:
Good point, done.